lib/jerboa/client.ex

```diff
@@ -121,6 +121,22 @@ defmodule Jerboa.Client do
   end
 
   @doc """
+  Creates permission on the allocation for the given peer
+  address
+
+  If permission is already installed for the given address,
+  the permission will be refreshed.
+
+  ## Example
+
+      iex> create_permission client, {192, 168, 22, 111}
+  """
+  @spec create_permission(t, peer :: ip) :: :ok | {:error, error}
```

Comment:
I think we could add one, since it doesn't hurt us at all and we can leave `create_permission/2` that doesn't require list as argument as override.